packages/strapi-plugin-graphql/services/Schema.js

```diff
@@ -205,14 +205,19 @@ const schemaBuilder = {
             : {};
 
           switch (type) {
-            case 'Mutation':
+            case 'Mutation': {
               // TODO: Verify this...
+              const [name, action] = acc[type][resolver].split('.');
```

Comment:
While this works for Mutation definitions such as
```
Mutation: {
    customMutation: {
        resolver: "Post.customMutation"
    }
}
```

This breaks when resolvers are objects (i.e. referencing an action within a plugin) such as...
```
Mutation: {
    customMutation: {
        resolver: {
            plugin: "my-plugin"
            handler: "Post.customMutation"
        }
    }
}
```

Admittedly, I'm not sure if it originally worked to begin with.